backend/gn_module_export/migrations/4cac712a2ce6_create_export_notifications.py

```diff
@@ -0,0 +1,129 @@
+"""Create export notifications
+
+Revision ID: 4cac712a2ce6
+Revises: c2d02e345a06
+Create Date: 2023-05-10 17:36:14.073847
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "4cac712a2ce6"
+down_revision = "c2d02e345a06"
+branch_labels = None
+depends_on = None
+
+
+CATEGORY_CODE = "EXPORTS-GENERATION"
```

Comment:
Par homogénéité avec IMPORT (https://github.com/PnX-SI/gn_module_import/pull/440/files), je le nommerai EXPORT-DONE (sans S).